telegram/ext/_chatjoinrequesthandler.py

```diff
@@ -43,17 +44,60 @@ async def callback(update: Update, context: CallbackContext)
 
             The return value of the callback is usually ignored except for the special case of
             :class:`telegram.ext.ConversationHandler`.
+        chat_id (:SCT[:obj:`int`], optional): Filters requests to allow only
+            those which are from a specified chat ID.
```

Comment:
This only mentions one chat id, we add usually do (s), so ID(s). And then without the a.